src/base/utils.js

```diff
@@ -85,6 +85,14 @@ export var Fullscreen = {
     } else if(document.msExitFullscreen) {
       document.msExitFullscreen()
     }
+  },
+  canGoFullscreen: function() {
+    return !(
+      document.fullscreenEnabled === false ||
+      document.webkitFullscreenEnabled === false ||
+      document.mozFullScreenEnabled === false ||
+      document.msFullscreenEnabled === false
+    )
```

Comment:
It's not as simple as that, because it has to fill all available viewport space and display on top of everything in the page. If some other element has an increased z-index, we should render on top of it as well.